### PR TITLE
Feature/admin creation

### PR DESCRIPTION
**Pull request by adheli**

### CODE REVIEW

**Review by seun-odebunmi — APPROVED** [review]
okay

File: src/main/java/ie/ait/bteam/drcare/data/model/User.java [hunk 1]
```diff
@@ -19,14 +19,14 @@
 
 	@Getter
 	@Setter
-	@Column
 	@NotNull
+	@Column(unique = true)
```
Comment (kenjao) [inline src/main/java/ie/ait/bteam/drcare/data/model/User.java]: I don't think Name should be unique

Comment (kenjao) [reply src/main/java/ie/ait/bteam/drcare/data/model/User.java]: Email should be unique instead

File: src/main/java/ie/ait/bteam/drcare/rest/controller/UserRestController.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
```
Comment (kenjao) [inline src/main/java/ie/ait/bteam/drcare/rest/controller/UserRestController.java:15]: I don't think there is need for Controller anymore

Comment (adheli) [reply src/main/java/ie/ait/bteam/drcare/rest/controller/UserRestController.java:15]: The @Controller annotation is for REST api working with @RequestMapping

Comment (kenjao) [reply src/main/java/ie/ait/bteam/drcare/rest/controller/UserRestController.java:15]: That's what the @RestController is for @adheli

**Review by kenjao — APPROVED** [review]

**Pull request merged**